test/questions.js

```diff
@@ -0,0 +1,133 @@
+import os from 'os';
+import test from 'ava';
+import path from 'path';
+import fs from 'fs-extra';
+import json from 'json-extra';
+import merge from 'lodash.merge';
+
+import getConfig from '../lib/getConfig';
+import questions, { choices } from '../lib/questions';
+import { withEmoji, withoutEmoji } from './fixtures/questions';
+
+const cwd = process.cwd();
+const date = new Date();
+const homedir = os.homedir();
+const fixtures = path.join(cwd, 'test', 'fixtures');
+const datetime = date.toISOString().slice(0, 10);
+const randomString = Math.random().toString(36).replace(/[^a-z]+/g, '').substr(0, 4);
+
+let globalExist = false;
+
+// rename global .sgcrc
+test.before(() => {
+  // rename global config
+  if (fs.existsSync(path.join(homedir, '.sgcrc'))) {
+    globalExist = true;
+    fs.renameSync(path.join(homedir, '.sgcrc'), path.join(homedir, `.sgcrc.${randomString}-${datetime}.back`));
+  }
+
+  // rename local sgcrc
+  fs.renameSync(path.join(cwd, '.sgcrc'), path.join(cwd, '.sgcrc_default'));
+});
+
+test.after.always(() => {
+  // rename global config
+  if (globalExist) {
+    fs.renameSync(path.join(homedir, `.sgcrc.${randomString}-${datetime}.back`), path.join(homedir, '.sgcrc'));
+  }
+
+  // rename local sgcrc
+  fs.renameSync(path.join(cwd, '.sgcrc_default'), path.join(cwd, '.sgcrc'));
+});
+
+test('choices are rendered without emoji', (t) => {
+  const sgc = getConfig(path.join(fixtures, '.sgcrc'));
+  const choicesList = choices(sgc);
+
+  t.deepEqual(choicesList, withoutEmoji);
+});
+
+test('choices with inherit mode | all', (t) => {
+  const sgc = getConfig(path.join(fixtures, '.sgcrc_inherit_all'));
+  const sgcrc = json.readToObjSync(path.join(cwd, '.sgcrc_default'));
+
+  t.deepEqual(sgc, merge({}, sgcrc, sgc));
```

Comment:
Yeah that is true, but what I understand is that, the source objects `sgcrc, sgc` are recursively added into the target object, and since your target object is a plain new object `{}` it should not take any effect. 

Correct me if I am wrong but if i  change everything to Object assign, every test gets passed quite fine.